### PR TITLE
10 new product page design

### PR DESCRIPTION
**Pull request by mrdigre**
@MiltonTulli check new product page styling

### CODE REVIEW

**Review by MiltonTulli — CHANGES_REQUESTED** [review]
Please include a proper description of your work. Don't tag me on the description but do request review through github. Since this pr is mainly ui would be nice to see some screenshots at the description. 
Also link properly the issue you pretend to close. 


Regarding your changes: 
- Page still doesn't look as expected. See issue description and code template included there. 
- "Add to cart" button looks bad (maybe you re behind main branch)
- Quantity selector is broken. 
- Page doesnt look good on mobile. 
Screenshot 2023-08-29 at 16 08 34
Screenshot 2023-08-29 at 16 08 23
Screenshot 2023-08-29 at 16 08 19

File: src/app/components/CartButton.tsx [hunk 1]
```diff
@@ -1,23 +1,65 @@
...
 const CartButton = ({ product }) => {
-  const { addToCart } = useCart(); //context
+  const { addToCart } = useCart(); // Context
+  const [quantity, setQuantity] = useState(1); // State to track quantity
 
   const handleAddToCart = () => {
-    addToCart(product);
-    window.alert("Product added succesfully")
+    addToCart(product, quantity); 
+    window.alert("Product added successfully");
```
Comment (MiltonTulli) [inline src/app/components/CartButton.tsx:11]: This notification should be inside addToCart function

File: src/app/products/[id]/page.tsx [hunk 2]
```diff
@@ -2,65 +2,74 @@
...
+                    <div className="mb-4">
+                        <span className="font-bold text-gray-700 mt-2 text-bg">Product Description:</span>
+                        <p className="text-gray-600 mt-2"> {fetchedProduct?.description} </p>
+                    </div>
+                        
+                    {/* <div className="flex items-center">
+                        
+                        <QuantityInput />
+                        
+                    </div> */}
```
Comment (MiltonTulli) [inline src/app/products/[id]/page.tsx:54]: Remove commented code

**Review by MiltonTulli — CHANGES_REQUESTED** [review]

**Comment by mrdigre** [comment]
- Product page styling modified considering the provided template:
image
- Add to Cart button was behind main branch.
image- 
- Quantity selector functionality fixed
- Mobile styling fixed.

Additionally worked on Cart styling on this branch.

File: src/app/components/Cart.tsx [hunk 3]
```diff
@@ -1,14 +1,34 @@
...
+    const showMessageIfEmptyCart = totalQuantity === 0
+
+    // href={route}
+    // key={route}
+    // className={classNames(
+    //   label.current
+    //     ? "bg-gray-900 text-white"
+    //     : "text-gray-300 hover:bg-gray-700 hover:text-white",
+    //   "rounded-md px-3 py-2 text-lg font-medium"
+
```
Comment (MiltonTulli) [inline src/app/components/Cart.tsx:18]: Remove this comments

**Review by MiltonTulli — APPROVED** [review]
Looks good to me

**Pull request merged**